qmail-rspawn.8

```diff
@@ -9,7 +9,11 @@ reads a series of remote delivery commands from descriptor 0,
 invokes
 .B qmail-remote
 to perform the deliveries,
-and prints the results to descriptor 1.
+and prints the results to descriptor 1. If the environment variable QMAILREMOTE
+is defined, the executable defined by QMAILREMOTE is invoked instead. This can
+be set to
+.B spawn-filter
+which can run the email message through a filter before calling qmail-local
```

Comment:
calling qmail-local here doesn't make sense, no?

Reply:
Yup. It should be qmail-remote

Reply:
Corrected and pushed